test.apl

```diff
@@ -18,6 +18,11 @@
   test¯warn test¯read_unicode filename
 ∇
 
+⍝ Evaluate APL code (as string) and return output (in original type) or error message (as string).
+∇z←test∆try apl_string
+  z←(↑¯1↑⎕EC apl_string)[1;]
```

Comment:
I may want to be a little smarter about detecting actual errors. I still need to find out more about APL's error handling.

Reply:
Yeah, I think I need to add a few things.